### ISSUE TITLE
Alternate link mechanism as NVDA "clickable" issue workaround

### ISSUE THREAD

**Issue by cleidigh** [microsoft/vscode#60120]
Having done a lot with accessibility and labeling for the settings editor the issue with in NVDA repeating clickable multiple times on links is incredibly distracting for the accessibility user.  The issue link below describes the variety of problems and  discussion about solutions but you can see there's no clear progress.

Should we have a discussion on some current workaround solutions?

https://github.com/nvaccess/nvda/issues/7430

cc: @roblourens

**Comment by isidorn** [comment]
@cleidigh reading the discussion it seems like it is fixed in chrome cannary in december 2017.
I am not sure what version of chrome is that, but I would put my money on that we still did not pick up that update on electron? What would be your estitimate @bpasero 
@cleidigh is there some easily reproducable issue on our side which we could try out with https://github.com/electron/fiddle and try out if it reproduces with newer electron versions?

**Comment by bpasero** [comment]
@cleidigh @isidorn just try it out with our exploration builds if the fix is in. If not, I would think we need to check with Electron 4 once we get there (next year).

Download:
* macOS: [Download](https://az764295.vo.msecnd.net/exploration/45498d93166483c2eff886e28f8dab915ae382fc/VSCode-darwin-exploration.zip)
* Linux: [Download](https://az764295.vo.msecnd.net/exploration/5a736c2e4904ed114fae2d5941508505f53afc66/code-exploration_1.28.0-1538127308_amd64.deb)
* Windows: [Download](https://az764295.vo.msecnd.net/exploration/5a736c2e4904ed114fae2d5941508505f53afc66/VSCodeUserSetup-x64-1.28.0-exploration.exe)

**Comment by cleidigh** [comment]
@bpasero 
@isidorn 
Cool, I tested the exploration built with electron three.  It does appear to behave better.  There are still duplicated "clickable" outputs but noticeably fewer.  If you looked at some of the issues, it's not a singular issue there are several triggers for the clickable messages.  I have to dig further but I think there still might be some changes needed to get to a single clickable message.

We cannot fight City Hall so I will look at if there's any low hanging fruit otherwise I think we just have to deal with it.

**Comment by isidorn** [comment]
@cleidigh I suggest you also try this https://github.com/electron/fiddle  it is pretty cool!
As for newever exploration builds @bpasero knows better

**Comment by cleidigh** [comment]
Electron Fiddle looks great.  I assume if I use both the same Electron/Chromium versions as VSC with this should be a pretty good representation of VSC, correct?

**Comment by isidorn** [comment]
I can not reproduce this issue any more with the latest vscode insiders which uses electron 4.
Closing
@cleidigh let me know if you still see it and I will reopen
